### PR TITLE
[Method Reference] LPS-167246 Convert method reference to ServiceTracker in module portal-messaging

### PR DESCRIPTION
**Pull request by W2jx**
Hi @dantewang , I have resent it.

Previous: https://github.com/liferay-core-infra/liferay-portal/pull/5010

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:lima

**Comment by W2jx** [comment]
ci:stop

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:lima

**Comment by zhangyan0701** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
no regressions found. SF issue is not related

**Comment by W2jx** [comment]
no conflicts（7/20 8:28）

File: modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/DefaultMessageBus.java [hunk 1]
```diff
@@ -201,120 +214,32 @@ public void removedService(
...
-
 		shutdown(true);
 
+		_destinationEventListenerServiceTracker.close();
+
 		for (Destination destination : _destinations.values()) {
 			destination.destroy();
 		}
 
 		_destinations.clear();
```
Comment (dantewang) [inline modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/DefaultMessageBus.java:238]: @W2jx Is this part still needed? When you close a `ServiceTracker`, all the tracked ones will be untracked, during which `removedService` will be called for each.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by W2jx** [comment]
Hi @dantewang , I have modified the code.

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
ci:test:lima

**Comment by catarineaguiar** [comment]
no regressions found

**Comment by W2jx** [comment]
no conflicts(7/24 8:21)

**Comment by dantewang** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/5073

**Pull request closed without merging**